vara/transport.go

```diff
@@ -30,6 +30,21 @@ func (m *Modem) DialURL(url *transport.URL) (net.Conn, error) {
 		}
 	}
 
+	// Select public
+	if err := m.writeCmd(fmt.Sprintf("PUBLIC ON")); err != nil {
```

Comment:
What does this do? Would be nice to have a small comment explaining the implications.

Reply:
I should have been more verbose indeed, this is however a gray area as there is no documentation. The modem accepts that command even though it is not documented. RMS Express uses it for all connections. It matches the default WinLink service code for Amateur Radio (PUBLIC). Pat currently does not have any support for configuring the service code though.